src/Servers/Kestrel/Core/src/Internal/Http3/Http3Connection.cs

```diff
@@ -332,12 +332,12 @@ private void UpdateStreamTimeouts(DateTimeOffset now)
                     Debug.Assert(streamDirectionFeature != null);
                     Debug.Assert(streamIdFeature != null);
 
-                    var context = CreateHttpStreamContext(streamContext);
```

Comment:
It's not generally clear why the changes in this file matter. Are they primarily about avoiding `CreateHttpStreamContext(...)` in the `else` portion of `CreateHttp3Stream<TContext>(...)`❔ (I'm somewhat unsure whether I'm missing another branch where `CreateHttpStreamContext(...)` is no longer called.)

Separately `context` and `_context` are both used below. Could we use slightly more distinct names❔